### PR TITLE
WEBUI-935: forbid access to admin tabs if user profile is non admin

### PR DESCRIPTION
**Pull request by swarnadipa-dev**

### CODE REVIEW

**Review by GuruprasadAnk1996 — DISMISSED** [review]

**Review by rakeshkumar1019 — DISMISSED** [review]

**Review by SnehalNaik-DEV — DISMISSED** [review]
Looks good.

File: elements/routing.js [hunk 1]
```diff
@@ -86,9 +86,36 @@ page('/admin/:tab?', (data) => {
...
+      app.currentUser.isAdministrator || app.currentUser.extendedGroups.find((grp) => grp.name === 'powerusers');
+    if (hasPermission) {
+      if (data.params.tab) {
+        app.selectedAdminTab = 'user-group-management';
+      }
+      app.show('admin', [data.params.type]);
+    } else {
+      app.showError(404, '', data.path);
+    }
+  });
```
Comment (rajatgoel-gl) [inline elements/routing.js:92]: Kindly remove repeatitive code

File: elements/routing.js [hunk 2]
```diff
@@ -86,9 +86,36 @@ page('/admin/:tab?', (data) => {
...
+      app.currentUser.isAdministrator || app.currentUser.extendedGroups.find((grp) => grp.name === 'powerusers');
+    if (hasPermission) {
+      if (data.params.tab) {
+        app.selectedAdminTab = 'user-group-management';
+      }
+      app.show('admin', [data.params.type, data.params.id]);
+    } else {
+      app.showError(404, '', data.path);
+    }
+  });
```
Comment (rajatgoel-gl) [inline elements/routing.js:98]: Kindly remove repeatitive code

**Review by rajatgoel-gl — CHANGES_REQUESTED** [review]

**Review by GuruprasadAnk1996 — APPROVED** [review]

**Review by Nisha-QA — APPROVED** [review]
Looks good

**Review by rajatgoel-gl — APPROVED** [review]

**Review by rakeshkumar1019 — APPROVED** [review]

**Pull request merged**